### DISCUSSION TITLE
Github Pages Envrionment Active

### DISCUSSION THREAD

**Discussion by aliraza5490** [community/community#60631] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I've disable GitHub pages on my repository and removed the workflow file as well which I was using for deployment to pages. But, I'm still having this github-pages active in Envrionment section. How to fix it?

**Comment by aliraza5490** [comment]
Fixed.

**Reply by opensusefan** [reply]
It would be much better if you also post the answer how did you fixed that as someone may had that same issue and can benefit from your answer + can save the time from asking for the same question.

**Comment by aliraza5490** [comment, marked as answer]
If you go to the repository settings. On the left side menu, you will find an Environment's option. There you can manage the environments for your repository.
